api/tests/app/test_app.py

```diff
@@ -0,0 +1,162 @@
+import pytest
+
+from api.models.app import AppDTO
+from api.tests.utils.common_functions import get_app_from_ini_config_file_json, json_multi_type_to_json_string, \
+    create_app_config, camel_case_to_snake_case_dict
+# The line below is absolutely necessary. Fixtures are passed as arguments to test functions. That is why IDE could
+# not recognized them.
+from api.tests.utils.fixtures_tests import config_rollback, app_config
+
+
+def expected_response_function(config_sample_path):
+    expected_response = get_app_from_ini_config_file_json(config_sample_path)
+    expected_response = json_multi_type_to_json_string(expected_response)
+
+    # It is weird that this change does not appear in response.json()
+    # Because if request was successfully, the PUT request will change the flag "has_been_configured"
+    expected_response["has_been_configured"] = "False"
+
+    return expected_response
+
+
+def expected_response_default():
+    """When no valid key is provided to the PUT request, config is updated with default values stored in AppDTO"""
+    expected_response = AppDTO().__dict__
+    expected_response = camel_case_to_snake_case_dict(expected_response)
+    expected_response = json_multi_type_to_json_string(expected_response)
+    expected_response["dashboard_url"] = expected_response["dashboard_u_r_l"]
+    del expected_response["dashboard_u_r_l"]
+    return expected_response
+
+
+# pytest -v api/tests/app/test_app.py::TestClassGetAppConfig
+class TestClassGetAppConfig:
+    """Get App Config, GET /app"""
+
+    def test_get_app_config(self, config_rollback):
+        client, config_sample_path = config_rollback
+
+        response = client.get('/app')
+
+        expected_response = get_app_from_ini_config_file_json(config_sample_path)
+        expected_response["dashboardurl"] = expected_response["dashboard_url"]
+        del expected_response["dashboard_url"]
+
```

Comment:
Why is this change required?

I tested the endpoint `/app` and it returns the key "dashboard_url"

Reply:
That is right, the endpoint returns he key "dashboard_url". However, my function get_app_from_ini_config_file_json, returns all the keys fine, less "dashboard_url", so we have to change it manually. Do not forget that i am changing "expected_response", not "response".